EDITORS.md

```diff
@@ -254,3 +280,52 @@ To configure indexing, pass a JSON hash as part of the initialization options fo
   }
 }
 ```
+
+# All initialization options
+
+Each LSP client can control various abilities of the LSP at startup. The following JSON dictionary contains all of the
+available initialization options. Generally, editor LSP clients will configure LSP servers using a dictionary in their
+configuration languages (JSON, Lua, ELisp, etc.). _NOTE: `enabledFeatures` may also be an array with feature names as
+strings rather than a string->bool dictionary as in this documentation.
```

Comment:
The array style is deprecated and we're only keeping it for backwards compatibility. Let's not advertise it.